### PR TITLE
Add 10.11 Studio Pro release notes

### PR DESCRIPTION
**Pull request by ConnorLand**

### CODE REVIEW

**Comment by atennapel** [comment]
On line 29: "In the Studio Preferences" should be "In the Studio Pro Preferences"

File: content/en/docs/releasenotes/studio-pro/10/10.11.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+* Newly created projects will be configured to use Java 21 by default.
+
+We recommend that you upgrade apps and Marketplace modules to Java 21.
+
+### Improvements
+
+* We changed the log level of the *Tasks cannot be executed anymore due to model changes* message in the task queue from **critical** to **error**.  (Ticket 199565)
+* We ensure that the `vendorlib-sbom.json` file does not change when dependencies do not change, reducing the number of git conflicts related to this file. (Ticket 208993)
+* We now pass a render mode to the pluggable widgets API to indicate which mode the page editor is in.
+* We added a `CanApplyJumpTo` attribute to the `System.Workflow` entity that indicates whether it is allowed to apply a jump-to option to the workflow in its current state.
```
Comment (MarkvanMents) [inline content/en/docs/releasenotes/studio-pro/10/10.11.md]: "whether it is allowed" - what is it? Does it mean:
```suggestion
* We added a `CanApplyJumpTo` attribute to the `System.Workflow` entity to indicate whether you can apply a jump-to option to the workflow in its current state.
```

File: content/en/docs/releasenotes/studio-pro/10/10.11.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+* We made some performance improvements when finding usages in protected modules.
+* We added a **Support Tools** submenu to the **Help** menu, this submenu contains the following tools: Performance logging, Profiling, and Optimize MPR.
```
Comment (MarkvanMents) [inline content/en/docs/releasenotes/studio-pro/10/10.11.md]: Documentation?

File: content/en/docs/releasenotes/studio-pro/10/10.11.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+* We improved parameter validation while trying REST requests in consumed REST service editor (Beta). Requests having parameter values with spaces in URL will now raise errors as they do when used in microflow action.
+* We upgraded react-navigation to version 6.x.x and extended navigation transitions options for Studio Pro.
```
Comment (MarkvanMents) [inline content/en/docs/releasenotes/studio-pro/10/10.11.md:43]: Should this not have a real version? **6.x.x** looks like a placeholder - I would think there is an actual version which is shipped?

File: content/en/docs/releasenotes/studio-pro/10/10.11.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+* We added rendering support for Markdown tables, code blocks, and headers in MendixChat.
+* We added a small delay before showing the MxAssist Logic Bot blue dot.
+* We fixed memory leaks that happened when closing and opening applications in Studio Pro.
+* We do not allow you to provide an invalid or duplicate request name in the consumed REST service document.
+* We made some performance improvements to several operations in Studio Pro, including renaming entities, pasting in the domain model editor, copying and duplicating documents, and importing packages.
+* We improved relay-outing in the logic editors: when adding objects to a loop, it only expands in the relevant direction(s).
+* We have made a couple of small performance improvements to the Logic Bot.
+* The menu items in the **View** menu are now ordered alphabetically and have individually assigned keyboard shortcuts.
+* We improved consistency checks for parameters in the Consumed REST Request (beta) microflow action. Parameters with different casing are also marked as duplicate now.
+* We added support for parameters in Path URL inside a Consumed REST service (beta) document.
```
Comment (MarkvanMents) [inline content/en/docs/releasenotes/studio-pro/10/10.11.md]: Are some of these UX elements?
```suggestion
* We improved consistency checks for parameters in the **Consumed REST Request** (beta) microflow action. Parameters with different casing are also marked as duplicate now.
* We added support for parameters in **Path URL** inside a **Consumed REST service** (beta) document.
```
There is also a capitalization difference between "REST Request" and "REST service"

File: content/en/docs/releasenotes/studio-pro/10/10.11.md [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+* When using MySQL as the database, setting a query timeout works again.
+* We resolved an issue in Studio Pro where an error occurred when attempting to open **Preferences** while creating a new app.
+* We fixed an issue where aborting a workflow resulted in an error if the workflow had an ongoing activity that had just finished.
+* We fixed an issue where a pluggable widget with a [selection property](/apidocs-mxsdk/apidocs/pluggable-widgets-property-types/#selection) linked to a [data source property](/apidocs-mxsdk/apidocs/pluggable-widgets-property-types/#datasource) which was not required resulted in an exception when the data source was not configured. 
+* We fixed an exception that happened upon closing a non-version-controlled app (after you previously opened a version controlled app and did not restart Studio Pro).
+* We fixed an issue in the [Core.RetrieveByPath](https://apidocs.rnd.mendix.com/10/runtime/com/mendix/core/Core.html) Runtime API, where having a new external object and retrieving over an association from the child's side resulted in an error.
```
Comment (MarkvanMents) [inline content/en/docs/releasenotes/studio-pro/10/10.11.md:91]: I am never sure what we mean by child. Do we mean the side which doesn't own the association?

Perhaps it is just me?

**Review by MarkvanMents — COMMENTED** [review]
Have made some proofing changes in https://github.com/mendix/docs/pull/7771/commits/0d4dbb12b9a895a430f0d3778c7e71eb8b3dcea1 and left a few comments.

But nothing which would prevent the doc being published.

**Review by MariaShaposhnikova — APPROVED** [review]

**Pull request merged**